samples/exemplar/build.gradle.kts

```diff
@@ -1,8 +1,15 @@
 plugins {
   kotlin("jvm")
+  application
+  id("com.github.johnrengelman.shadow")
```

Comment:
why do we need this / what does this do?

Reply:
I can probably remove this, I thought it was necessary to run the sample service but the `application` plugin is probably sufficient.

Reply:
Removed!